### PR TITLE
fixed height and icons positioning issues

### PR DESCRIPTION
**Pull request by muganwas**

### CODE REVIEW

**Comment by mericda** [comment]
@muganwas tested this, thanks for fixing it! One more issue that will be good to address: Can we make the top bar/part of this modal persistent to scroll? 
CleanShot 2022-01-05 at 21 40 58@2x

**Comment by muganwas** [comment]
> @muganwas tested this, thanks for fixing it! One more issue that will be good to address: Can we make the top bar/part of this modal persistent to scroll? CleanShot 2022-01-05 at 21 40 58@2x

Sure.

File: src/components/play/layer-settings/LayerListPopup.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const LayerListPopup = ({
     user,
     round
 }) => (
-    <Box className={showMixerPopup ? classes.mixerPopup : classes.hidden}>
+    <Box className={showMixerPopup ? classes.mixerPopup : classes.hidden} style={height < 400 ? { height: height - 142, top: -(height - 138) } : {}}>
```
Comment (choshun) [inline src/components/play/layer-settings/LayerListPopup.js:22]: oof, this looks a bit brittle, but if it works it works I guess

File: src/App.js [hunk 2]
```diff
@@ -42,6 +42,15 @@ function App ({ setUser, setRounds, setIsShowingSignInDialog }) {
             active: '#EAEAEA'
           }
         },
+        breakpoints: {
```
Comment (choshun) [inline src/App.js:45]: awesome! thank yooooou!!!!!

File: src/components/play/layer-settings/LayerSettings.js.old [hunk 3]
```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react'
```
Comment (choshun) [inline src/components/play/layer-settings/LayerSettings.js.old:1]: huh, keep on seeing this file added 🤔 , maybe it's another .old file?

**Review by choshun — APPROVED** [review]
Thanks for adding the breakpoints!

File: src/components/play/layer-settings/LayerListPopup.js (hunk 1 above)
Comment (muganwas) [reply src/components/play/layer-settings/LayerListPopup.js:22]: Could be done in a better way I guess, I'll clean it up a little.

File: src/components/play/layer-settings/LayerSettings.js.old (hunk 3 above)
Comment (muganwas) [reply src/components/play/layer-settings/LayerSettings.js.old:1]: please just ignore these for now, I'll get rid of them very soon.

**Pull request merged**